### PR TITLE
Fix feature flaw

### PR DESCRIPTION
**Pull request by vijay-shankaranand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ![Ui](docs/images/Ui.png)
 
 * This is our team project for the course **CS2103T** in AY2023/2024 Semester 1.<br>
-* The project simulates an ongoing software project for a desktop application (called _JobFestGo_) used for managing contact details of the relevant personnels for a job fest.
+* The project simulates an ongoing software project for a desktop application (called _JobFestGo_) used for managing contact details of the relevant Contactnels for a job fest.
```
Comment (tjch-o) [inline README.md]: spelling mistake and it shouldn't be capitalised

File: docs/UserGuide.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ Deletes the specified tag from JobFestGo.
 Format: `delete_tag t/TAG_NAME`
 
 * Deletes the tag with the specified tag name `TAG_NAME`.
-* The tag name **must be an existing tag** vendor, personal, customer, …​
+* The tag name **must be an existing tag** vendor, Contactal, customer, …​
```
Comment (tjch-o) [inline docs/UserGuide.md]: merging mistake

File: docs/tutorials/AddRemark.md [hunk 3]
```diff
@@ -326,31 +326,31 @@ After the previous step, we notice a peculiar regression — we went from di
...
+    public static final String MESSAGE_DELETE_REMARK_SUCCESS = "Removed remark from Contact: %1$s";
 //...
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        List<Person> lastShownList = model.getFilteredPersonList();
+        List<Contact> lastShownList = model.getFilteredContactList();
 
         if (index.getZeroBased() >= lastShownList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+            throw new CommandException(Messages.MESSAGE_INVALID_Contact_DISPLAYED_INDEX);
```
Comment (tjch-o) [inline docs/tutorials/AddRemark.md:344]: contact should be capitalised to match the rest of the words

File: src/main/java/seedu/address/model/ModelManager.java [hunk 4]
```diff
@@ -21,12 +21,12 @@
 import seedu.address.model.task.TaskInReminderPredicate;
 
 /**
- * Represents the in-memory model of the address book data.
+ * Represents the in-memory model of the Job Fest Go data.
```
Comment (tjch-o) [inline src/main/java/seedu/address/model/ModelManager.java:24]: should this be split?

**Review by tjch-o — CHANGES_REQUESTED** [review]
generally lgtm! with some nits to fix... this was definitely a very tedious job especially remembering to update all the diagrams good job!

File: docs/tutorials/TracingCode.md [hunk 5]
```diff
@@ -177,7 +177,7 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
 
 1. Stepping through the method shows that it calls `ArgumentTokenizer#tokenize()` and `ParserUtil#parseIndex()` to obtain the arguments and index required.
 
-1. The rest of the method seems to exhaustively check for the existence of each possible parameter of the `edit` command and store any possible changes in an `EditPersonDescriptor`. Recall that we can verify the contents of `editPersonDesciptor` through the 'Variables' window.<br>
+1. The rest of the method seems to exhaustively check for the existence of each possible parameter of the `edit` command and store any possible changes in an `EditContactDescriptor`. Recall that we can verify the contents of `editContactDesciptor` through the 'Variables' window.<br>
```
Comment (Khaleelur-Rahman) [inline docs/tutorials/TracingCode.md:180]: should `editContactDescriptor` be `EditContactDescriptor` also?

File: docs/tutorials/TracingCode.md [hunk 6]
```diff
@@ -195,20 +195,20 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
...
+       Contact editedContact = createEditedContact(contactToEdit, editContactDescriptor);
+       if (!contactToEdit.isSameContact(editedContact) && model.hasContact(editedContact)) {
+           throw new CommandException(MESSAGE_DUPLICATE_Contact);
        }
-       model.setPerson(contactToEdit, editedContact);
-       model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-       return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedContact));
+       model.setContact(contactToEdit, editedContact);
+       model.updateFilteredContactList(PREDICATE_SHOW_ALL_CONTACTS);
+       return new CommandResult(String.format(MESSAGE_EDIT_Contact_SUCCESS, editedContact));
```
Comment (Khaleelur-Rahman) [inline docs/tutorials/TracingCode.md:205]: capitalisation

File: src/main/java/seedu/address/logic/commands/contact/EditContactCommand.java [hunk 7]
```diff
@@ -126,35 +128,35 @@ public boolean equals(Object other) {
...
         return new ToStringBuilder(this)
                 .add("index", index)
-                .add("editPersonDescriptor", editPersonDescriptor)
+                .add("editContactDescriptor", editContactDescriptor)
                 .toString();
     }
 
     /**
      * Stores the details to edit the contact with. Each non-empty field value will replace the
      * corresponding field value of the contact.
```
Comment (Khaleelur-Rahman) [inline src/main/java/seedu/address/logic/commands/contact/EditContactCommand.java:144]: should there be a line break after `the`?

File: src/main/java/seedu/address/model/UserPrefs.java [hunk 8]
```diff
@@ -100,21 +100,21 @@ public boolean equals(Object other) {
...
-        return Objects.hash(guiSettings, addressBookFilePath, tagListFilePath, eventListFilePath);
+        return Objects.hash(guiSettings, jobFestGoFilePath, tagListFilePath, eventListFilePath);
     }
 
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Gui Settings : " + guiSettings);
-        sb.append("\nLocal address book data file location : " + addressBookFilePath);
+        sb.append("\nLocal job fest go data file location : " + jobFestGoFilePath);
```
Comment (Khaleelur-Rahman) [inline src/main/java/seedu/address/model/UserPrefs.java:117]: Single word `jobFestGo`

**Review by Khaleelur-Rahman — CHANGES_REQUESTED** [review]
LGTM! Just a few minor nits.

File: src/main/java/seedu/address/logic/commands/contact/EditContactCommand.java (hunk 7 above)
Comment (vijay-shankaranand) [reply src/main/java/seedu/address/logic/commands/contact/EditContactCommand.java:144]: did not change the comments

File: docs/tutorials/TracingCode.md (hunk 5 above)
Comment (vijay-shankaranand) [reply docs/tutorials/TracingCode.md:180]: followed convention

**Pull request merged**